reactive/kubernetes_master.py

```diff
@@ -1944,11 +1964,11 @@ def configure_apiserver():
 
     admission_plugins = [
         'PersistentVolumeLabel',
+        'PodSecurityPolicy',
+        'NodeRestriction'
     ]
 
     auth_mode = hookenv.config('authorization-mode')
-    if 'Node' in auth_mode:
```

Comment:
I think this is safe. The NodeRestriction admission controller is documented [here](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction). It's just going to prevent kubelet from modifying Node or Pod resources that belong to other kubelets. Kubelet should never try to do that anyway.

Reply:
Thanks!